spikeinterface/sortingcomponents/benchmark/benchmark_clustering.py

```diff
@@ -179,15 +191,91 @@ def run(self, peaks=None, positions=None, method=None, method_kwargs={}):
         self.gt_labels = self.gt_sorting.to_spike_vector()['unit_ind']
 
 
+    def _get_colors(self, sorting, excluded_ids=[-1]):
+        from spikeinterface.widgets import get_unit_colors
+        colors = get_unit_colors(sorting)
+        result = {}
+        for key, value in colors.items():
+            result[sorting.id_to_index(key)] = value
+        for key in excluded_ids:
+            result[key] = 'k'
```

Comment:
Here there is a misleading between unit_ind and unit_id.
You are using the index in the first loop but then you use the id (-1 is an id).
Clarrify this.